addons/partner_autocomplete/models/res_partner.py

```diff
@@ -124,12 +126,21 @@ def enrich_company(self, company_domain, partner_gid, vat, timeout=15):
 
     @api.model
     def read_by_vat(self, vat, timeout=15):
+        check_vies_res = check_vies(vat)
+        del check_vies_res['requestDate']
+        vies_result = {res: check_vies_res[res] for res in check_vies_res}
```

Comment:
What is the point ?

Reply:
check_vies returns an object of type `checkVatResponse`
and I would get this error `TypeError: Object of type checkVatResponse is not JSON serializable`
so I had to turn it into a dict

Reply:
Ok, what about `vies_result = dict(check_vies_res)` or `vies_result = {**check_vies_res}` ?

Reply:
For `{**check_vies_res}` I got `TypeError: 'checkVatResponse' object is not a mapping`
And dictionary creation also failed